tests/performance/compare-results.js

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies.
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { median } = require( './utils' );
+
+/**
+ * Parse test files into JSON objects.
+ *
+ * @param {string} fileName The name of the file.
+ * @returns An array of parsed objects from each file.
+ */
+const parseFile = ( fileName ) =>
+	JSON.parse(
+		fs.readFileSync( path.join( __dirname, '/specs/', fileName ), 'utf8' )
+	);
+
+// The list of test suites to log.
+const testSuites = [ 'home-block-theme', 'home-classic-theme' ];
+
+// The current commit's results.
+const testResults = Object.fromEntries(
+	testSuites.map( ( key ) => [
+		key,
+		parseFile( `${ key }.test.results.json` ),
+	] )
+);
+
+// The previous commit's results.
+const prevResults = Object.fromEntries(
+	testSuites.map( ( key ) => [
+		key,
+		parseFile( `before-${ key }.test.results.json` ),
+	] )
+);
+
+const args = process.argv.slice( 2 );
+
+const summaryFile = args[ 0 ];
+
+function formatAsMarkdownTable( rows ) {
+	let result = '';
+	const headers = Object.keys( rows[ 0 ] );
+	for ( const header of headers ) {
+		result += `| ${ header } `;
+	}
+	result += '|\n';
+	for ( const header of headers ) {
+		result += '| ------ ';
+	}
+	result += '|\n';
+
+	for ( const row of rows ) {
+		for ( const value of Object.values( row ) ) {
+			result += `| ${ value } `;
+		}
+		result += '|\n';
+	}
+
+	return result;
+}
+
+let summaryMarkdown = `**Performance Test Results**\n\n`;
+
+if ( process.env.GITHUB_SHA ) {
+	summaryMarkdown += `Performance test results for ${ process.env.GITHUB_SHA } are in 🛎️!\n\n`;
```

Comment:
```suggestion
	summaryMarkdown += `🛎️ Performance test results for ${ process.env.GITHUB_SHA } are in!\n\n`;
```

Reply:
Personal preference, but I feel the emoji at the beginning breaks up the section a bit.